src/chia_log/handlers/wallet_added_coin_handler.py

```diff
@@ -18,11 +21,10 @@ class WalletAddedCoinHandler(LogHandlerInterface):
     def config_name() -> str:
         return "wallet_added_coin_handler"
 
-    def __init__(self, config: Optional[dict] = None):
+    def __init__(self, config: confuse.core.Configuration):
```

Comment:
This is rather a `confuse.core.ConfigView` right? I wonder how we can make it more readable without having to parse all the call-sites to determine the full nesting.